devTools/check_trailing_whitespaces.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eu
+
+files_with_trailing_whitespaces=$(
+    find . \
+        -type f \
+        -not -path "./.git/*" \
+        -not -path "./vendor/*" \
+        -exec grep -EIHn "\\s$" {} \;
+)
+
+if [[ "$files_with_trailing_whitespaces" ]]
+then
+    printf '\033[97;41mTrailing whitespaces detected:\033[0m\n'
+    e=$(printf '\033')
```

Comment:
I'm not sure to understand why the `e` here: what does it represent or why is it necessary?

Reply:
Used in the next line it seems, as a shortcut.